node-src/tasks/build.ts

```diff
@@ -122,7 +122,8 @@ export const buildStorybook = async (ctx: Context) => {
 
     const subprocess = execaCommand(ctx.buildCommand, {
       stdio: [null, logFile, null],
-      preferLocal: true,
+      preferLocal: false, // When true, this will run in the node version set
+                          // by the action (node20), not the version set in the workflow
```

Comment:
```suggestion
      // When true, this will run in the Node version set by the
      // action (node20), not the version set in the workflow.
      preferLocal: false,
```